lib/type.gi

```diff
@@ -145,3 +145,90 @@ function ( type )
     fi;
     return res;
 end);
+
+InstallGlobalFunction( TypeOfOperation,
+function(op)
+    local type, flags, types, catok, repok, propok, seenprop,
+          t, res;
+    if not IsOperation(op) then
+        ErrorNoReturn("<op> must be an operation");
```

Comment:
Almost everywhere else, we say `<oper>`. I'd adjust at the very least the error message, but perhaps also rename the argument?

Point is, if I see `oper` anywhere in the GAP code, I immediately know what it means/is.